papers.html

```diff
@@ -159,24 +159,66 @@ <h5>Kruckow et al. 2024</h5>
+      </div>
+    </div>
+  <hr/>
+</div>
+
+<div class="container p-4">
+  <h1><a href="https://arxiv.org/abs/2311.07528" target="_blank" style="color: rgb(0,0,0)">Wind Roche-lobe Overflow in Low-Mass Binaries: Exploring the Origin of Rapidly Rotating Blue Lurkers</a></h1>
+  <p></p>
+  <h5>Sun et al. 2024, accepted in ApJ</h5>
+  <p></p>
+
+  <p>
+    <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Sun20240621" aria-expanded="false" aria-controls="Sun20240621">
+      Details
+    </button>
+  </p>
+    <div class="collapse" id="Sun20240621">
+      <div class="card card-body">
+        <p><b>Authors: </b> Meng Sun, Sasha Levina, Seth Gossage, Vicky Kalogera, Emily M. Leiner, Aaron M. Geller, Zoheyr Doctor</p>
+        <p><b>Access: </b> <a href="https://arxiv.org/abs/2311.07528" target="_blank">arXiv:2311.07528</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024ApJ...969....8S/abstract" target="_blank">NASA ADS</a></p>
+        <p><b>Abstract: </b>Wind Roche-Lobe Overflow (WRLOF) is a mass-transfer mechanism proposed by Mohamed and Podsiadlowski (2007) for stellar binaries wherein the wind acceleration zone of the donor star exceeds its Roche lobe radius, allowing stellar wind material to be transferred to the accretor at enhanced rates. WRLOF may explain characteristics observed in blue lurkers and blue stragglers. While WRLOF has been implemented in rapid population synthesis codes, it has yet to be explored thoroughly in detailed binary models such as \texttt{MESA} (a 1D stellar evolution code), and over a wide range of initial binary configurations. We incorporate WRLOF accretion in \texttt{MESA} to investigate wide low-mass binaries at solar metallicity. We perform a parameter study over the initial orbital period and stellar mass. In most of the models where we consider angular momentum transfer during accretion, the accretor is spun up to the critical (or break-up) rotation rate. Then we assume the star develops a boosted wind to efficiently reduce the angular momentum so that it could maintain a sub-critical rotation. Balanced by boosted wind loss, the accretor only gains $\sim 2\%$ of its total mass, but can maintain a near-critical rotation rate during WRLOF. Notably, the mass-transfer efficiency is significantly smaller than in previous studies in which the rotation of the accretor is ignored. We compare our results to observational data of blue lurkers in M67 and find that the WRLOF mechanism can qualitatively explain the origin of their rapid rotation, their location on the HR diagram and their orbital periods. </p>
```

Comment:
```suggestion
        <p><b>Abstract: </b>Wind Roche-Lobe Overflow (WRLOF) is a mass-transfer mechanism proposed by Mohamed and Podsiadlowski (2007) for stellar binaries wherein the wind acceleration zone of the donor star exceeds its Roche lobe radius, allowing stellar wind material to be transferred to the accretor at enhanced rates. WRLOF may explain characteristics observed in blue lurkers and blue stragglers. While WRLOF has been implemented in rapid population synthesis codes, it has yet to be explored thoroughly in detailed binary models such as MESA (a 1D stellar evolution code), and over a wide range of initial binary configurations. We incorporate WRLOF accretion in MESA to investigate wide low-mass binaries at solar metallicity. We perform a parameter study over the initial orbital period and stellar mass. In most of the models where we consider angular momentum transfer during accretion, the accretor is spun up to the critical (or break-up) rotation rate. Then we assume the star develops a boosted wind to efficiently reduce the angular momentum so that it could maintain a sub-critical rotation. Balanced by boosted wind loss, the accretor only gains &sim;2% of its total mass, but can maintain a near-critical rotation rate during WRLOF. Notably, the mass-transfer efficiency is significantly smaller than in previous studies in which the rotation of the accretor is ignored. We compare our results to observational data of blue lurkers in M67 and find that the WRLOF mechanism can qualitatively explain the origin of their rapid rotation, their location on the HR diagram and their orbital periods. </p>
```

Reply:
thanks, updated